dockerclient.go

```diff
@@ -258,6 +258,37 @@ func (client *DockerClient) ContainerChanges(id string) ([]*ContainerChanges, er
 	return changes, nil
 }
 
+func (client *DockerClient) ContainerStats(id string, stopChan <-chan struct{}) (<-chan StatsOrError, error) {
+	uri := fmt.Sprintf("/%s/containers/%s/stats", APIVersion, id)
```

Comment:
Def. agree we should do it in a separate PR.  For that change we should just tag the repo at 1.5 and move on.  If someone wants the older version, they can pull the tag.  How about a compromise for this PR until we get the library in better shape by just commenting it (and maybe ListNetworks too) so we don't forget.

The flip side of this is it looks really weird in the UCP (and presumably DTR) logs that says the API version is 1.15.